src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -764,32 +831,53 @@ def visit_FunCall(self, node: gtir.FunCall) -> IteratorExpr | ValueExpr:
         assert isinstance(node.type, ts.ScalarType)
         dtype = dace_utils.as_dace_type(node.type)
 
-        return self._get_tasklet_result(dtype, tasklet_node, "result")
+        return self._construct_tasklet_result(dtype, tasklet_node, "result")
+
+    def visit_FunCall(self, node: gtir.FunCall) -> IteratorExpr | ValueExpr:
+        if cpm.is_call_to(node, "deref"):
+            return self._visit_deref(node)
+
+        elif cpm.is_call_to(node, "neighbors"):
+            return self._visit_neighbors(node)
+
+        elif cpm.is_applied_reduce(node):
+            return self._visit_reduce(node)
+
+        elif cpm.is_applied_shift(node):
+            return self._visit_shift(node)
+
+        elif isinstance(node.fun, gtir.SymRef):
+            return self._visit_builtin(node)
+        else:
+            raise NotImplementedError
```

Comment:
There are two things here, you should add a bit more specific, what have you encountered.

My second point is more off topic.
For my understanding `raise NotImplementedError` throws the class _object `NotImplementedError` and not an instance of this class.
So would you not have to write `raise NotImplementedError()` to create an instance?